Add canConsecutiveWalljump throughout Zebes

kjbranch opened the pull request:
- Update the description to be anywhere that requires 3+ consecutive wall jumps.
- It is now required for canSunkenDualWallClimb and canInsaneWalljump.

osse101 commented on region/maridia/inner-yellow.json:
```diff
@@ -879,7 +879,8 @@
                   "notable": false,
-                  "requires": []
+                  "requires": [ "canConsecutiveWalljump" ],
+                  "devNote": "This should require something more to kill the Choot."
```
I would say `canPreciseWallJump` or `Screw Attack` or `Power Bomb` or `enemy hit`
There is a fair sized safe spot on the ledge, and you can continue walljumping when it floats towards that side.
Wave is too gimicky with the camera manip you have to do.
Firing a strong weapon just before landing would be a precise jump and some kind of quickFire tech

osse101 commented on region/maridia/inner-yellow.json:
```diff
@@ -1704,6 +1706,10 @@
                   "notable": false,
                   "requires": [
+                    {"or": [
+                      "HiJump",
```
HiJump+Walljump ?

osse101 commented on region/norfair/crocomire.json:
```diff
@@ -1664,7 +1666,10 @@
                     "canMellaIceClip",
-                    "canWalljump"
+                    {"or":[
+                      "HiJump",
```
HiJump+walljump ?

osse101 commented on region/norfair/crocomire.json:
```diff
@@ -1624,7 +1624,8 @@
                     "canManipulateMellas",
-                    "HiJump"
+                    "HiJump",
+                    "canConsecutiveWalljump"
```
This only takes 1 walljump, might even be able to do it with none.

kjbranch replied:
I think it should take 2 wall jumps to get onto the Mella and then one more to get from the Mella to the item.
https://youtu.be/a02RaagZqjo

osse101 replied:
You linked the no HiJump version.
With hijump its jump up to the bug then jump into walljump.

kjbranch replied:
Well, that's embarrassing.

osse101 commented on tech.json:
```diff
@@ -320,7 +320,10 @@
-                      "requires": [ "canDelayedWalljump" ],
+                      "requires": [
+                        "canDelayedWalljump",
+                        "canConsecutiveWalljump"
```
`canInsaneWallJump` is better left describing the jump itself.  Many cases are 2 jumps total, taco tank is 1 walljump.

kjbranch replied:
I was planning on changing this to canStaggeredWalljump once that tech is merged in. I figure it would take a couple simple wall  jumps to position Samus. Taco Tank and Screw CWJ  don't need that though ....

osse101 commented on tech.json:
```diff
@@ -340,7 +343,7 @@
                 "canWalljump"
               ],
-              "note" : "Climbing a wall with several consecutive walljumps.",
+              "note" : "Climbing a wall with three or more consecutive walljumps, where a single failure will require restarting.",
```
Failure is a strange descriptor since missing a walljump means you fall a bit and try again a little lower.  Accidently breaking spin could be the failure at this skill level; an enemy hit doesn't seem meant to be included in this meaning.

kjbranch commented on region/crateria/central.json:
```diff
@@ -573,7 +573,13 @@
+                  "requires": [
+                    "canDelayedWalljump",
+                    {"or": [
+                      "HiJump",
```
Although the Delayed Walljumps are a little tighter, only having to do one is much easier for a beginner. Varia Randomizer classifies this strat as Very Hard with and Hardcore without Hi Jump: https://randommetroidsolver.pythonanywhere.com/presets

kjbranch commented:
I think I've addressed all of your concerns here. 
- I will update some of them to canStaggeredWalljumps once #523 is merged.
- I will add these tech to WRITG type ledges in another pull request
- I will add these tech to Lava Dive strats after #529 is merged
